### PR TITLE
Python.Activities: Fix threading model for load/execute script

### PR DESCRIPTION
**Pull request by florinakke**
Some modules ( like xlwings) perform COM initialization ( https://docs.microsoft.com/en-us/windows/win32/api/combaseapi/nf-combaseapi-coinitializeex ); this fails since the threading model is already set on MTA. Ensure that Load/Exec script are executed on STA threads.

### CODE REVIEW

**Review by cosmyo — APPROVED** [review]
Tested with the project provided internally.

File: Activities/Python/UiPath.Python/Impl/Engine.cs [hunk 1]
```diff
@@ -124,7 +124,7 @@ public Task Release()
         public Task<PythonObject> LoadScript(string code, CancellationToken ct)
         {
             ct.ThrowIfCancellationRequested();
-            return Task.Run(() =>
+            return RunSTA((cancellationToken) =>
```
Comment (LeifLazarUiPath) [inline Activities/Python/UiPath.Python/Impl/Engine.cs:127]: No need to pass ct to the func, it can use cancellationToken from the caller.

**Review by LeifLazarUiPath — CHANGES_REQUESTED** [review]

File: Activities/Python/UiPath.Python/Impl/Engine.cs (hunk 1 above)
Comment (florinakke) [reply Activities/Python/UiPath.Python/Impl/Engine.cs]: fixed; also combined the commits to a single one.

**Review by LeifLazarUiPath — APPROVED** [review]

**Comment by grigoremircea** [comment]
Deployed to Connect Feed (https://gallery.uipath.com/api/v2) as version "1.0.7594.23824"

**Comment by grigoremircea** [comment]
Correction deployed as version v1.1.7594.35963 on Connect feed.

**Comment by theloginerror** [comment]
This pull request has been mentioned on **UiPath Community Forum**. There might be relevant details there:

https://forum.uipath.com/t/load-python-script-error-when-running-python-script/200446/14

**Pull request merged**